packages/amplify-provider-awscloudformation/src/push-resources.ts

```diff
@@ -555,6 +555,33 @@ export const updateStackForAPIMigration = async (context: $TSContext, category:
   }
 };
 
+/**
+ * Publish files that Amplify Studio depends on outside the zip file so that can read
+ * without sreaming from the zip.
+ */
+const uploadStudioBackendFiles = async (s3: S3, bucketName: string) => {
+  const amplifyDirPath = pathManager.getAmplifyDirPath();
+  const studioBackendDirName = 'studio-backend';
+  // Delete the contents of the studio backend directory first
+  await s3.deleteDirectory(bucketName, studioBackendDirName);
+  // Create a list of file params to upload to the deployment bucket
+  const uploadFileParams = [
+    'cli.json',
+    'amplify-meta.json',
+    'backend-config.json',
+    'schema.graphql',
+    'transform.conf.json',
+    'parameters.json',
+  ]
+    .flatMap(baseName => glob.sync(`**/${baseName}`, { cwd: amplifyDirPath }))
+    .filter(filePath => !filePath.startsWith('backend/'))
+    .map(filePath => ({
+      Body: fs.createReadStream(path.join(amplifyDirPath, filePath)),
+      Key: path.join(studioBackendDirName, filePath.replace('#current-cloud-backend/', '')),
```

Comment:
Do all of these path operations work on Windows? I'm not sure if the forward slash will be handled properly

Reply:
Good catch. No need for the slashes as `path.join` normalizes. `glob` modules takes care of the os abstraction as well and requires forward slashes. Will rev for removing the others.